apps/console/src/index.jsp

```diff
@@ -276,6 +276,38 @@
                     return applicationDomain.replace(/\/+$/, '') + getTenantPath();
                 }
 
+                /**
+                 * Construct the auth params for organization login `authorize` requets.
+                 *
+                 * @remarks This only applies to the new authz runtime.
+                 *
+                 * @returns {string} Contructed auth params.
+                 */
+                function getAuthParamsForOrganizationLogins(orginalPrams) {
+                    if (startupConfig.legacyAuthzRuntime) {
+                        return orginalPrams;
+                    }
+
+                    var authParams = Object.assign({}, orginalPrams);
+
+                    if (getOrganizationPath()) {
+                        var initialUserOrgInLocalStorage = localStorage.getItem("user-org");
+                        var orgIdInLocalStorage = localStorage.getItem("org-id");
+
+                        if (orgIdInLocalStorage) {
+                            if (orgIdInLocalStorage === getOrganizationName() && initialUserOrgInLocalStorage !== "undefined") {
```

Comment:
Will it be okay to use es6 features in JSP?

Reply:
+1. Will be fixed in a followup PR.

Reply:
Several `const` were there from previous commits.

Removed in https://github.com/wso2/identity-apps/pull/5141/commits/444205565a5f13d6aaac2e7326e0f1393f590c70